StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/GenericInfoScreen/GenericInfoFooterView.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  GenericInfoFooterView.swift
+//  StripeFinancialConnections
+//
+//  Created by Krisjanis Gaidis on 7/2/24.
+//
+
+import Foundation
+import UIKit
+
+func GenericInfoFooterView(
+    footer: FinancialConnectionsGenericInfoScreen.Footer?,
+    didSelectPrimaryButton: @escaping () -> Void,
+    didSelectSecondaryButton: @escaping () -> Void,
+    didSelectURL: @escaping (URL) -> Void
+) -> UIView? {
+    if let footer {
```

Comment:
Another minor nit pick; I would lean toward using a `guard` here as well, but it comes down to personal preference!

Reply:
I think that's very fair because the existence of `footer` is a sort-of precondition for the whole function, so the addition of a guard handles the "precondition" in a more clear manner (and reduces extra tab/pyramid-of-doom)

changed it: https://github.com/stripe/stripe-ios/pull/3745/commits/6afecb56b03b15b4d90d085971f7f59fbb1fddb4